README.md

```diff
@@ -7,20 +7,31 @@
 
 ## Inputs
 
-### `i`
+### `input`
 
 The path to a pip requirements file, default is "./requirements.txt"
 
 Be sure to quote paths with spaces.
 
-### `o`
+### `output`
 
 Output filename, default is "./bom.xml"
 
 Be sure to quote paths with spaces.
 
+### `format`
```

Comment:
:x:  there is no such incoming param in current https://github.com/CycloneDX/gh-python-generate-sbom/blob/master/action.yml